### PR TITLE
CFY 4929 deployment update

### PR DESCRIPTION
**Pull request by ran-z**

### CODE REVIEW

File: tests/workflow_tests/test_deployment_update.py [hunk 1]
```diff
@@ -2004,3 +2012,104 @@ def test_remove_deployment(self):
         self.assertEquals(len(deployment_update_list), len(undel_depups_ids))
         for i in deployment_update_list.items:
             self.assertIn(i['id'], undel_depups_ids)
+
+
+class TestDeploymentUpdateMisc(DeploymentUpdateBase):
```
Comment (max-orlov) [inline tests/workflow_tests/test_deployment_update.py]: So there's this class: TestDeploymentUpdateMixedOperations...

Comment (ran-z) [inline tests/workflow_tests/test_deployment_update.py]: I figured mixed operations are about mixing of specific updates, e.g. node and relationship update; this class is about misc update scenarios. do you still think they should be unified?

File: tests/testenv/es_schema_creator.py [hunk 2]
```diff
@@ -32,10 +32,22 @@
 DEPLOYMENT_SCHEMA = {
     'deployment': {
         'properties': {
-            'plan': {
```
Comment (max-orlov) [inline tests/testenv/es_schema_creator.py]: so the plan was never really there to put in storage?

Comment (ran-z) [inline tests/testenv/es_schema_creator.py]: at the dark old times, we used to have the deployment plan under the deployment object; this was later removed - but apparently it was forgotten in this module :)

File: tests/workflow_tests/test_deployment_update.py (hunk 1 above)
Comment (max-orlov) [inline tests/workflow_tests/test_deployment_update.py]: well, maybe the name is misleading, but the mixoperations basically held any test which is more than one step/skipping install/executing custom workflow etc...

it sounds like it's a good idea to extract any non step related test to your new class...

File: tests/testenv/es_schema_creator.py (hunk 2 above)
Comment (max-orlov) [inline tests/testenv/es_schema_creator.py]: cool beans

File: tests/workflow_tests/test_deployment_update.py (hunk 1 above)
Comment (ran-z) [inline tests/workflow_tests/test_deployment_update.py]: ok, i moved all but two tests to my new class; kinda difficult seeing that on github comparison view but you can try and just view the file itself

**Pull request merged**